serverless/installing_serverless/installing-knative-serving.adoc

```diff
@@ -6,19 +6,19 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-After you install the {ServerlessOperatorName}, you can install Knative Serving by following the procedures described in this guide.
-
-This guide provides information about installing Knative Serving using the default settings. However, you can configure more advanced settings in the KnativeServing custom resource definition.
-
-For more information about configuration options for the KnativeServing custom resource definition, see xref:../installing_serverless/serverless-install-config-options.adoc#serverless-install-config-options[Advanced installation configuration options].
+After you install the {ServerlessOperatorName}, you can install Knative Serving.
+This guide provides information about installing Knative Serving using the default settings. However, you can configure more advanced settings in the `KnativeServing` custom resource definition.
```

Comment:
Could do a "custom resource definition (CRD)" here so that you can just say "CRD" in the next sentence that follows, if ye wanted.